### DISCUSSION TITLE
Current conditions always has null values

### DISCUSSION THREAD

**Discussion by sekarb** [weather-gov/api#629] [Q&A]
Hi,
I am trying to get current conditions for a station in my case, I am testing with SFOC1 with the url
https://api.weather.gov/stations/SFOC1/observations/latest

I always get null for everything with quality control being 'Z'
Am I doing anything wrong?

I have searched the similar issues on here. Looks like there was some report like this in 2021 and comment said it was an upstream MADIS issue and they are working to fix it. Has it not been fixed still? Is there a workaround to get the current conditions? I have tried just the '/observations' endpoint and still get null values

On the weather.gov website for the station I see it shows proper numbers. So the website is getting it from somewhere.

Thank you

**Comment by dwhitemv25** [comment]
According to Mesowest, SFOC1 is a HADS station. I spot-checked another HADS station here in Oregon, SERO3, and it has no data in the API either, so it may be an issue with that mesonet specifically. 

Many HADS stations only have a few meteorological sensors, as their mission is hydrological monitoring. (They likely have water level/flow sensors that don't import to MADIS.) ASOS and RAWS stations tend to be more reliable sources of weather (meteorological) data.
